src/main/java/com/github/timm/cucumber/generate/ClassNameGenerator.java

```diff
@@ -1,37 +1,38 @@
-package com.github.timm.cucumber.generate;
```

Comment:
It looks like you're using a different formatter.

What IDE are you using?
I am using Eclipse + Default Eclipse formatter rules.

Please could you reconcile - it makes reviewing much easier (I realise I haven't defined a standard so far)